ruskeyverbs/templates/ruskeyverbs/answer_evaluation.html

```diff
@@ -11,20 +14,43 @@ <h2 class="incorrect">Incorrect</h2>
 <p>The correct answer is: <span class="bold">{{ answer }}</span></p>
 <p class="center">{{ russian_text }}</p>
 <h3>Score: {{ score }}</h3>
+{% if score <= 90 %}
+  <p>Retype the correct answer (with punctuation and capitalization):</p>
+  <input type="text" name="" v-model="userinbox" value="" v-on:keyup='CompareAns'></input>
```

Comment:
Is keyup the right event here? Seems like potentially a lot of back and forth with the server. Maybe v-on="blur:..." would be more appropriate?